org/pr/installable-build.ts

```diff
@@ -117,31 +109,17 @@ async function getDownloadCommentText(status) {
   const artifacts = await circleCIArtifacts(status)
 
   const commentJsonArtifact = artifacts.find(artifact => artifact.path.endsWith("comment.json"))
-
   if (commentJsonArtifact) {
-    // <Debug: Remove me>
-    console.log(
-      `Comment JSON - '${commentJsonArtifact.url}'`
-    )
-    // </Debug: Remove me>
-    
     // Download the JSON file so we can get the comment text
-    const res = await fetch(commentJsonArtifact.url)
+    const commentJsonArtifactUrl = `${commentJsonArtifact.url}?circle-token=${CIRCLECI_TOKEN}`
```

Comment:
After our discussion yesterday – this appears to be where all the magic happens. LGTM!

Reply:
Yep, it is :-)